### DISCUSSION TITLE
mmcv.cnn.bricks.registry

### DISCUSSION THREAD

**Discussion by sidiangongyuan** [open-mmlab/mmengine#1356] [Q&A]
When I want to use :
from mmcv.cnn.bricks.registry import (ATTENTION,
                                      TRANSFORMER_LAYER,
                                      TRANSFORMER_LAYER_SEQUENCE)
 it cannot find the reference . So I think this package has been removed. So can I find the part in mmengine？

**Comment by fanqiNO1** [comment]
Sure, you can find them in `mmengine.registry.MODELS`. https://github.com/open-mmlab/mmengine/blob/53474ef1ba0b166508c231fa525b55b580adf20f/mmengine/registry/root.py#L31

**Reply by sidiangongyuan** [reply]
Thank you. Can you give me an example to use ATTENTION or other models?

**Reply by karltan0328** [reply]
Have you found it?😭

**Reply by sidiangongyuan** [reply]
Sorry, I did not find it.  I used another version of the old one instead

**Reply by karltan0328** [reply]
I read the requirement.txt and use another version too.😂

**Comment by cht619** [comment]
..

**Comment by wenjie710** [comment]
Not solved yet :(

**Comment by Yandong-Luo** [comment]
Not solved yet :(
